src/components/ScoreResult/index.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import {
+  StyledScoreResult,
+  ScoreResultGroup,
+  ScoreResultGroupHeader,
+  Score,
+} from './styled';
+import { usePrevious } from '../../helpers';
+
+const ScoreResult = ({
+  score,
+  actualScore,
+  color,
+  reversed,
+  iterationInterval,
+}) => {
+  const [transitioning, setTransitioning] = useState(false);
+  const previousActualScore = usePrevious(actualScore);
+
+  useEffect(() => {
+    if (
+      !transitioning &&
+      previousActualScore !== null &&
+      previousActualScore !== actualScore
+    ) {
+      setTransitioning(true);
+    }
```

Comment:
Add: 
if (score === actualScore && !transitioning) {
      setAnswerColor('Green');
    }

Reply:
The way I am suggesting this does have a bug - it changes the background while the score is still being counted. Will update with workaround when I find one.

Reply:
Work-around if you add in the phase state as a prop. It doesn't change the background until the phase is set to 'results'
if (phase === 'results' && score === previousActualScore) {
      setAnswerColor('Green');
    }

Reply:
I added in an animating box-shadow. Let me know if it works!